### ISSUE TITLE
Allow single point coordinates in .get_timespan

### ISSUE THREAD

**Issue by dbrownf** [agrobioinfoservices/ClimMobTools#6]
The current version only allows to input geographic coordinates as bounding boxes and not as single point coordinates.

**Comment by kauedesousa** [comment]
This problem is solved in `ClimMobTools` v0.2-8. See code bellow:

````
library("ClimMobTools")

lonlat <- data.frame(lon = runif(10, 11, 12),
                     lat = runif(10, 55, 58))

pdates <- as.integer(runif(10, 17666, 17670))
pdates <- as.Date(pdates, origin = "1970-01-01")

rainfall(lonlat[1,],
         day.one = pdates[1],
         span = 50)

````


Regarding the call of a single point instead of a bounding box. This is something that we have to deal with `nasapower` developers, since this is how function `nasapower::get_power` works, from which `ClimMobTools::rainfall` and `ClimMobTools::temperature` depends on.

**Comment by dbrownf** [comment]
Thanks Kaue.

Actually `nasapower::get_power` accepts both formats: bounding box or single points. [https://power.larc.nasa.gov/docs/v1/#latloncoords](https://power.larc.nasa.gov/docs/v1/#latloncoords)

I usually get from the data with function `nasapower::get_power` using single point format, so that's how I realized that `ClimMobTools `only accepts bounding box format in v0.2-7.

It seems to me that the new code (v0.2-8) transforms a single point to a bounding box. If that is the case, then the problem with the limited extent of a bounding box will remain. I think this will not be a problem if the coordinates are provided as single points. 

Please let me know if it is a misunderstanding from my side.

**Comment by kauedesousa** [comment]
You're right. But these functions were build with the main idea to work with many points, considering the crowdsourcing philosophy behind ClimMob. Off course it should work with one single point.  I will keep this issue open until I solve it in the upcoming version of ClimMobTools. 

I also have this issue open in another section (here https://github.com/agrobioinfoservices/ClimMobTools/issues/5), since I intend to optimise the `.get_timespan` to be faster.
